e2e/playwright/command-bar-tests.spec.ts

```diff
@@ -487,4 +487,53 @@ test.describe('Command bar tests', { tag: ['@skipWin'] }, () => {
       await toolbar.expectFileTreeState(['main.kcl', 'test.kcl'])
     })
   })
+
+  test(`Can add a named parameter or constant`, async ({
```

Comment:
good that this is testing a "hard case", referencing existing var.

If we're able to resolve https://github.com/KittyCAD/modeling-app/pull/5840/files#r1999768352 probably should add a test for it too.

Reply:
I added unit tests for the base logic does that work or should I add an E2E test too?

Reply:
Yeah that seems fine.